views/layouts/layout-pane.njk

```diff
@@ -8,16 +8,35 @@
   <div class="app-o-pane__content">
     <main id="content" class="app-c-content" role="main">
       <div class="app-c-content__header">
-        <h1 class="govuk-heading-xl">
+
+        {% if status %}
```

Comment:
There's a lot of duplication here. Could we look at moving the status tag inside the h1 so that the margins are applied correctly and adding additional styling for it? That way we'd avoid having to wrap the whole thing in an if/else and duplicating the markup.

Reply:
Have fixed this specific issue